js/darkmode.js

```diff
@@ -0,0 +1,45 @@
+onload = onload();
+const icon = document.getElementById('icon');
+icon.onclick = function () {
+  const element = document.body;
+  element.classList.toggle('dark-theme');
+  if (element.classList.contains('dark-theme')) {
+    icon.src = 'img/sun.png';
+    setCookie('theme', 'dark', 30);
+  } else {
+    icon.src = 'img/moon.png';
+    setCookie('theme', 'light', 30);
+  }
+};
+
+function onload() {
+  const themeIcon = document.getElementById('icon');
+  if (getCookie('theme') === 'light') {
+    document.body.classList.remove('dark-theme');
+    themeIcon.src = 'img/moon.png';
+  } else if (getCookie('theme') === 'dark') {
+    document.body.classList.add('dark-theme');
+    themeIcon.src = 'img/sun.png';
+  }
+}
+
+function setCookie(name, value, days) {
+  var expires = '';
+  if (days) {
+    var date = new Date();
+    date.setTime(date.getTime() + days * 24 * 60 * 60 * 1000);
+    expires = '; expires=' + date.toUTCString();
+  }
+  document.cookie = name + '=' + (value || '') + expires + '; path=/';
```

Comment:
add the domain too here `domain=.realdevsquad.com`

Reply:
let's make use of template strings here for better readability and use.

Reply:
Can you please give a example here ??

Reply:
@rohan09-raj Please refer the following example:
``` document.cookie = `${name}=${value}; expires=${expires}; domain=${domain}; path=/`;```
Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
> @rohan09-raj Please refer the following example: `` document.cookie = `${name}=${value}; expires=${expires}; domain=${domain}; path=/`; `` Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Thank you